display_list/benchmarking/dl_complexity_metal.cc

```diff
@@ -329,6 +329,11 @@ void DisplayListMetalComplexityCalculator::MetalHelper::drawDiffRoundRect(
   AccumulateComplexity(complexity);
 }
 
+void DisplayListMetalComplexityCalculator::MetalHelper::drawRoundSuperellipse(
+    const DlRoundSuperellipse& rse) {
+  // TODO(dkwingsmt): implement
```

Comment:
Just FYI you do not need to implement this, as the complexity calculator is irrelevant for Impeller and never used.

Reply:
Ah, I see. Therefore I _should_ leave this unimplemented since RSE is not supported in Skia.

Reply:
I would vote for leaving it unimplemented.